fc/parsing/actions.py

```diff
@@ -1002,6 +1060,21 @@ def _convert_output_units(self):
             output_variables.update(variables)
         return output_variables
 
+    def _list_parameters(self):
+        """Populates the list of model parameters (inputs with a constant RHS)."""
+
+        for var in self.inputs:
+            try:
+                variable = self.model.get_variable_by_ontology_term(var.rdf_term)
+            except KeyError:
+                # Skip optional variables
```

Comment:
I think until this is implemented properly, we should be raising rather than skipping here.

Reply:
In this case I think the skip is what we'll want? The code in `_add_input_variables` could do with raising something maybe?

Reply:
Possibly. I guess we'll find out for sure when we implement optional handling!